runtime/j9vm/jvm.c

```diff
@@ -3856,13 +3856,13 @@ JVM_FindLibraryEntry(void* handle, const char *functionName)
 
 	Trc_SC_FindLibraryEntry_Entry(handle, functionName);
 
-#if defined(WIN32)
+#if defined(WIN32) /* defined(WIN32) */
```

Comment:
Isn't this redundant?

Reply:
true. fixed.

Reply:
The if should not be commented, else, elif and endif should be.